cmd/docgen/model_parameter.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 The SQLFlow Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"sqlflow.org/sqlflow/pkg/sql/codegen/xgboost"
+)
+
+func main() {
+	fmt.Print(`# Model Parameter Document
```

Comment:
`sqlflow.org` will generate indexes using document titles. Should have meaningful titles so the document for tensorflow, xgboost, alps wouldn't be the same.

Reply:
Sorry about the confusion. I want this file contains all the document for for tensorflow, xgboost, alps.

I have updated the description in https://github.com/sql-machine-learning/sqlflow/issues/1028 to the following.

> A command-line `docgen` in `cmd/docgen` that will call `func DocGenInMarkdown() string` for each package and combine them to `doc/model_parameters.md`

Reply:
I see